src/crypto/openssl/base64.h

```diff
@@ -83,8 +91,7 @@ namespace crypto
       auto err = ERR_get_error();
       if (err != 0)
       {
-        char err_str[256];
-        ERR_error_string(err, err_str);
+        auto err_str = OpenSSL::error_string(ERR_get_error());
         throw std::logic_error(fmt::format(
           "OSSL: Could not encode update to base64 string: {}", err_str));
```

Comment:
While we're here, we should make this exception more verbose, to help identify the issue if it repros only on the CI. ie - print `size`, `max_size`, `chunk_len`, maybe `data`, etc.